### PR TITLE
Translation Chinese Documents

### PR DESCRIPTION
**Pull request by VisualSJ**
@natural-law Please review this pr. Thanks

### CODE REVIEW

File: cocos/ui/UIHelper.h [hunk 1]
```diff
@@ -41,48 +41,92 @@ namespace ui {
...
      *
-     * @param root      The be searched root widget.
-     * @param name      The widget name.
-     * @return Widget isntance pointer.
+     * @~chinese 
+     * 查找根节点内特定name字段的子节点（部件）
+     * 此搜索将递归所有子节点（部件）。
+     * 
+     * @param root @~english The be searched root widget.
+     * @~chinese 被搜索的跟节点.
```
Comment (natural-law) [inline cocos/ui/UIHelper.h]: 错别字“根节点”

File: cocos/ui/UIImageView.h [hunk 2]
```diff
@@ -37,7 +37,8 @@ NS_CC_BEGIN
 namespace ui {
     class Scale9Sprite;
 /**
- * @brief A widget to display images.
+ * @brief @~english A widget to display images.
+ * @~chinese �?个小部件显示图像�?
```
Comment (natural-law) [inline cocos/ui/UIImageView.h]: 乱码了

**Comment by natural-law** [comment]
@VisualSJ 
Please solve the errors I commented.

**Comment by VisualSJ** [comment]
@natural-law please = =...review.... Thanks

**Comment by natural-law** [comment]
@andyque 
It's ok for merge.

**Pull request merged**